torch/_subclasses/fake_tensor.py

```diff
@@ -312,6 +314,7 @@ def __init__(self, fake_mode, elem, device: Union[torch.device, str]):
         assert device.type != "meta"
         self.fake_device = device
         self.fake_mode = fake_mode
+        self.has_sym_ints = symbolic_shapes.has_symbolic_sizes_strides(elem)
```

Comment:
we probably want to re-query every time right, because this is mutable, e.g. `resize_(...)`